latest/index.bs

```diff
@@ -236,9 +237,6 @@ Additional fields for the entry depend on "type" and are defined by the column `
 | `translation` | one of: `"translation":List[float]`, `"path":str` | translation vector, stored either as a list of floats (`"translation"`) or as binary data at a location in this container (`path`). The length of vector defines number of dimensions. |
 | `scale`       | one of: `"scale":List[float]`, `"path":str` | scale vector, stored either as a list of floats (`scale`) or as binary data at a location in this container (`path`). The length of vector defines number of dimensions. |
```

Comment:
It looks like this table still isn't rendered correctly:
![table-rendering](https://user-images.githubusercontent.com/4263537/151953662-f66aceba-74b4-41fb-bd49-ec5b27a0e083.png)
 I don't understand why. Could someone with more bikeshed experience give this a shot? cc @joshmoore @will-moore @sbesson

Reply:
I think I tried this way back on the previous PR, but didn't get much joy. I think @joshmoore suggested using an HTML table?